apps/web/components/ConnectWalletButton.tsx

```diff
@@ -191,6 +195,37 @@ export function ConnectWallet() {
                                 Disconnect
                               </Button>
                             </Menu.Item>
+                            {/* TODO: for prod lint to a */}
+
+                            <div className="flex gap-2 items-center justify-end border-t-[1px] border-neutral-soft pt-4">
+                              <a
+                                href="https://www.gardens.fund/"
```

Comment:
In order to go with same git version of the landing page
```suggestion
                                href="/"
```